site/en/blog/passkeys-on-icloud-keychain/index.md

```diff
@@ -0,0 +1,185 @@
+  <figcaption>Chrome's passkey dialog to sign in with a passkey saved to Chrome profile.</figcaption>
+</figure>
+
+If authentication is requested using form autofill (Conditional UI), passkeys
+from iCloud Keychain are listed as part of the autofill suggestions if the user
+has granted Chrome the necessary permission. If there are passkeys stored in the
+Chrome profile, they are suggested too.
+
+<figure>
+  {% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/MHCnKhNhwHM8x6eqrtsT.png", alt="Form autofill suggests passkeys from both iCloud Keychain and the Chrome profile.", width="800", height="535" %}
+  <figcaption>Form autofill suggests passkeys from both iCloud Keychain and the Chrome profile.</figcaption>
+</figure>
+
+## Saving passkeys to Chrome profile
+
+Even with iCloud Keychain support, users can choose to save passkeys to their
+Chrome profile by default .
+
+1.  In Chrome, visit `chrome://password-manager/settings`
+2.  Turn off the **Use passkeys across your Apple devices** toggle to explicitly
+    store passkeys to the Chrome profile by default.
+
+<figure>
+  {% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/QYeEKKTyg3TLXxpy41ow.png", alt="The user can choose to store passkeys to Chrome profile (only available locally and will not sync).", width="800", height="122" %}
+  <figcaption>The user can choose to store passkeys to Chrome profile (only available locally and will not sync).</figcaption>
+</figure>
+
+Users can also cancel the macOS's user verification dialog and choose iCloud
+Keychain or Chrome profile to save a new passkey.
+
+<figure>
+  {% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/XSJAYPiUuUJUbfNJ8XHs.png", alt="If the user cancels the dialog, Chrome asks to choose how to create a passkey.", width="800", height="780" %}
+  <figcaption>If the user cancels the dialog, Chrome asks to choose how to create a passkey.</figcaption>
+</figure>
+
+{% Aside %}
+
+On macOS 13.4 or earlier, all passkeys created on Chrome on macOS are stored to
+the [Chrome profile](https://support.google.com/chrome/a/answer/9025411) locally
+by default and they are not synchronized across devices. The stored passkeys are
+visible to the user from `chrome://settings/passkeys`.
+
+Even on macOS 13.5 or later, passkeys are stored to Chrome profile if the user
+is not signed into iCloud Drive.
+
+{% endAside %}
+
+## Distinguishing the origin of a passkey
+
+Again, there are no changes required from developers to align their passkey
+implementation with iCloud Keychain. The API behavior between Chrome profile and
+iCloud Keychain are already identical.
+
+However, relying parties can distinguish the origin of a passkey whether it's
+created on iCloud Keychain or on Chrome profile if they wish, by looking at the
+[AAGUID](https://www.w3.org/TR/webauthn/#aaguid) (an identifier indicating the
+type of the authenticator) in [the credential's authenticator
+data](https://www.w3.org/TR/webauthn-3/#sctn-attestation).
```

Comment:
```suggestion
Relying parties can access the origin of a passkey and distinguish whether it's
been created on iCloud Keychain or on Chrome profile, by looking at the
[AAGUID](https://www.w3.org/TR/webauthn/#aaguid) (an identifier indicating the
type of the authenticator) in [the credential's authenticator
data](https://www.w3.org/TR/webauthn-3/#sctn-attestation).
```

Reply:
What do you mean by "access the origin of a passkey" in this case?

Reply:
I think it's the equivalent of  "know the origin by looking at AAGUID". Do you think "access" is not appropriate because it might imply some sort of API call or some other reason? We can also just say "relying parties can find out the origin of a passkey".

Reply:
Yeah, I thought "access" and "the origin" might sound as if there's a website or an endpoint. "relying parties can find out the origin of a passkey" sounds better.

Reply:
Agreed, we don't want any confusion, thanks for raising it!